### DISCUSSION TITLE
Follower issues android 15 pixel 8

### DISCUSSION THREAD

**Discussion by Banken85** [NightscoutFoundation/xDrip#3745] [Bug reports] [answered]
Hi, I just bought a pixel 8 couple of weeks ago and install android 15 by the start of the week. Xdrip+ works perfectly fine for around 10-12 hours as a follower, then it stops getting new data from master and the only solution I find so far is to uninstall and install xdrip+ again. I read some people using pixel 8 is experiencing some issues with dexcom's sensors, mine is libre2 but only as a follower, not as a master. Xdrip have all the permissions allowed, battery usage unrestricted and moreover works fine for the first 10 hours. Does anyone know what could be wrong?
Thank you 
![Screenshot_20241101-234915](https://github.com/user-attachments/assets/2d659bd3-ea1c-4396-a499-6c1410fd672d)
![Screenshot_20241101-235022](https://github.com/user-attachments/assets/389123dc-8537-4aa3-93ff-77c739955628)
![Screenshot_20241101-235059](https://github.com/user-attachments/assets/fe7e9c72-5497-4dad-93a2-78f0ec00c165)
![Screenshot_20241101-235208](https://github.com/user-attachments/assets/672975f8-8977-40d4-8467-27288276bea8)

**Comment by Navid200** [comment]
Can you post a screenshot of the classic status page?

**Reply by Banken85** [reply]
Sure, here it is 
![Screenshot_20241101-235702](https://github.com/user-attachments/assets/969ff945-0781-4611-8996-681b41edadbc)

**Reply by Navid200** [reply]
Please post a screenshot of the following page:
`Settings` -> `xDrip Sync Settings`

**Reply by Banken85** [reply]
Here it is Navid

**Reply by Navid200** [reply, marked as answer]
Enable the setting on that page shown below both on the master and follower:  
![382423162-0d64dd10-3e73-4a0b-b679-413709d69899](https://github.com/user-attachments/assets/3019a5ce-8668-4d19-bd79-0071da653c5e)  

Then, please report if the problem persists.

**Reply by Navid200** [reply]
I'm so sorry I should have asked you to blackout the key.  I deleted the image from your post for that reason.

If you want to update the key, you can manually change it both on the master as well as the follower.  Alternatively, you can delete the key from the master.  After you leave and return to the page, you will see a new random key.  You will then need to update the follower to match.
I'm sorry about that.

**Reply by Banken85** [reply]
It works! Thank you so much Navid! You made my day :D

**Reply by Navid200** [reply]
You're welcome.

Please monitor and make sure it continues to work beyond 10-12 hours.  If it doesn't please report.

Thanks

**Reply by Banken85** [reply]
Dear Navid, after 2 days I have no more issues. Thanks a lot for your quickness and great work. You can close this thread and mark it as solved ☺️

**Reply by Navid200** [reply]
Great!  Thanks for the feedback.
We can leave discussions open.  But, it is great to have stated that all is good now.
